### ISSUE TITLE
Create PTR records from zone file

### ISSUE THREAD

**Issue by jr-RXcode** [octodns/octodns#146]
Feature request:
Have the ability to create a PTR record file from an existing zone file. When managing a zone file it would be nice to run a command that would generate a PTR record automatically in a new zone file. using octodns.

**Comment by poblahblahblah** [comment]
Hey @jr-RXcode,

We had this need and so I wrote a quick and dirty python script to get this done for us: https://gist.github.com/poblahblahblah/7f2b058bb5f408fd12b63161849500d3

It's pretty rough and very "scripty", but it gets the job done for our internal DNS needs since everything is carved up into a /24.

The "forward" zone configs that have a PTR generated look like this:
```
'dns-master1':
  ptr: /24
  ttl: 60
  type: A
  value: 192.168.168.10
```

If there are no other records for 192.168.168.0/24 it will create a new zone file containing the following record:
```
'10':
  ttl: 300
  type: PTR
  value: dns-master1.wc1.example.com.
```

We call the script using a Makefile we wrote.

Hope this is useful!
